### ISSUE TITLE
Make porting even easier

### ISSUE THREAD

**Issue by Programmiernutte** [scotws/TaliForth2#148]
Hi, 
while porting TaliForth2 to steckOS, I encountered two things that I had to work around in order to get TaliForth 2 running. 

1. While platform-*.asm contains .org $8000, taliforth.asm, which is included from there, contains .advance $8000. Because in my case, init_kernel is not run from any interrupt vector, I have to call it implicitly, either by putting the code in front of .require "../taliforth.asm" or put a jmp init_kernel there. In both cases, the assembler stops at that .advance $8000, because I am already at $8003. I suggest omitting the .advance $8000.
2. Interrupts. The routine "xt_cold" does a SEI to disable all interrupts. This is a problem on steckOS, where display of the text console relies on the blank interrupt. I suggest not disabling interupts there. If interrupt should be enabled or disabled is IMHO to be decided within the platform-*.asm file, as this is really a platform specific thing.

**Comment by SamCoVT** [comment]
Thank you, Programmiernutte, for submitting an issue.  

We are currently making some changes to try to make Tali Forth 2 more portable between platforms by moving more of the platform-specific addressing into platform-*.asm and eliminating fixed addressing in other locations, so your input is very helpful in our efforts.  My hardware has a similar memory map to the simulator we are using, and I'm only running Tali2 on my hardware, so I didn't catch the issue you are referring to in item 1.

I will take a look at implementing your suggestions, and I initially agree with both of them and think they should be simple fixes.  I just recently switched to interrupt-driven receive so I could use hardware handshaking, and I found that SEI myself just a little while ago.
